rotkehlchen/db/dbhandler.py

```diff
@@ -2267,6 +2270,15 @@ def edit_exchange(
                 self.conn.rollback()
                 raise InputError(f'Could not update DB user_credentials_mappings due to {str(e)}') from e  # noqa: E501
 
+        location_is_binance = location in (Location.BINANCE, Location.BINANCEUS)
+        if location_is_binance and binance_markets is not None:
+            try:
+                account_name = new_name if new_name is not None else name
```

Comment:
Replace "account_name". It's a confusing name for this variable. This is the exchange name.

```suggestion
                exchange_name = new_name if new_name is not None else name
```